.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET_KEY = 'sudgksagdjhsagdjhsaghjgda'
```

Comment:
Need to gitignore this file. This defeats the main purpose of a .env file (aside from keeping config out of code). No keys in source control/public domain ever

Reply:
🤦‍♀️